Network/Socket/Options.hsc

```diff
@@ -481,6 +500,12 @@ instance Storable StructLinger where
         (#poke struct linger, l_linger) p linger
 #endif
 
+-- | Existential socket option value
```

Comment:
I feel that "existential" is too academic to understand for many users.
How about this?
"A data type to hold several different socket option types such as Int and StructLinger".
Please feel free to fix my broken English.

Reply:
Ah yes, thanks, I was also a bit worried about that. What you wrote sounds great, I'll update this!